ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -907,3 +955,164 @@ objects:
           fi
           sleep 60 & wait
         done
+
+    # The setup-packet and test containers need libnns_wrapper to use ssh
+    # TODO(derekh): investigate if it can be added to that container images
+    - name: nss-wrapper-hack
+      image: ${IMAGE_LIBVIRT_INSTALLER}
+      terminationMessagePolicy: FallbackToLogsOnError
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -xe
+        cp /bin/mock-nss.sh /usr/lib64/libnss_wrapper.so /tmp/shared/
+    # We need to have a seperate setup container for packet.net servers
+    # as we need an image with terrafrom
+    - name: setup-packet
```

Comment:
Same as above: Am I right to think we would be running this pod on every e2e install, whether it is needed or not?

Reply:
See previous comment on CLUSTER_TYPE guard

Reply:
yeah so it would run and return 0 so the `setup-packet` junit test I mentioned above would pass on non-packet e2e-tests. makes sense.